expirationd.lua

```diff
@@ -327,48 +390,202 @@ end
 -- ========================================================================= --
 -- Expiration daemon management functions
 -- ========================================================================= --
+--
+-- {{{ Module methods
+--
+
+--- Module methods
+--
+-- @section Functions
```

Comment:
In my understanding a method accepts an argument like `self` or `this`. So I think that 'functions' would be more appropriate term for those pieces of code.

Reply:
Fixed.

```lua
--- a/expirationd.lua
+++ b/expirationd.lua
@@ -391,10 +391,10 @@ end
 -- Expiration daemon management functions
 -- ========================================================================= --
 --
--- {{{ Module methods
+-- {{{ Functions
 --
 
---- Module methods
+--- Functions
 --
 -- @section Functions
 
@@ -924,4 +924,4 @@ return {
     show_task_list = show_task_list_obsolete,
 }
 
--- }}} Module methods
+-- }}} Functions
```